content/en/docs/reference/command-line-tools-reference/feature-gates/local-storage-capacity-isolation-fs-quota-monitoring.md

```diff
@@ -6,14 +6,13 @@ _build:
   render: false
 
 stages:
-  - stage: alpha
+  - stage: beta
     defaultValue: false
-    fromVersion: "1.15"
+    fromVersion: "1.31"
```

Comment:
you are supposed to add new elements about the beta stage rather than replacing the existing alpha stage data. Please refer to other feature gates for the correct format.